demisto_sdk/commands/split/ymlsplitter.py

```diff
@@ -97,14 +98,16 @@ def extract_to_package_format(self) -> int:
         except ValueError as ex:
             print_error(str(ex))
             return 1
-        self.print_logs("Starting migration of: {} to dir: {}".format(self.input, output_path), log_color=LOG_COLORS.NATIVE)
+        self.print_logs("Starting migration of: {} to dir: {}".format(self.input, output_path),
+                        log_color=LOG_COLORS.NATIVE)
         output_path.mkdir(parents=True, exist_ok=True)
         base_name = output_path.name if not self.base_name else self.base_name
         code_file = output_path / base_name
         self.extract_code(code_file)
```

Comment:
Instead of having the try-except block on line 248 ([here](https://github.com/demisto/demisto-sdk/pull/2168/files#diff-913a538463e23a378949fb612aea23e4156a05d20f5922ca84e8e1ff412d0aedR248)) let's just add a condition to check if we are looking at addling or parsing